utilities/generate_messages_catalog.py

```diff
@@ -291,6 +281,30 @@ def _build_id_messages(python_module):
     return id_messages
 
 
+def _get_message_codes(msg_code_arg):
+    """
+    Get the correct message codes based on instance type of msgCodeArg
+
+    :param msg_code_arg: the current arg to pull msgCodes from
+    :type msg_code_arg: :class:`~ast.AST`
+    :return: the correct message code to use
+    :rtype: str
```

Comment:
you're right, it's not just a string, but is it a set? looks more like a tuple...